README.md

```diff
@@ -155,30 +159,157 @@ You can find the AWS repo push commands under your repository in ECR by selectin
-3. Navigate to the Task Definition, select the appropriate task definition name and create a new **_revision_** (note: **_not_** a new task definition)
-4. Find the Container Section and update the <version\> of the image to your new image version. No other values need to change.
-5. After saving the task definition, select the Task Definition name and choose Deploy->Update Service
-6. In the Update Service configuration select Force New Deployment and ensure the following are set:
+An email invite will be sent to the selected email address along with a one-time password which is valid for 10 days.
 
-    - Revision : matches the task definition revision you just created
-    - Desired Tasks : 1
-    - Min Running Tasks: 100%
-    - Max Running Tasks: 200%
+### Updating the running service using Terraform
 
-    Click on Update to trigger the deployment.
+If the application has been modified and the changes do not require the Cognito authentication or S3 store to be removed and re-staged (i.e _most_ application level changes) then the following can be performed:
 
-The deployment will take a couple of minutes, you should see two tasks running temporarily (the new one and the existing running task). Once the new task is deployed the original task container will drain and stop.
+- Build a new version of the container image and upload to ECR as per the instructions earlier in this guide.
+- Change directory to the **service terraform**
 
-Your service should now be running the new image.
+  ```bash
+  cd terraform/service
+  ```
+
+- In the appropriate environment variable file env/sandbox/sandbox.tfvars, env/dev/dev.tfvars or env/prod/prod.tfvars
+  - Change the _container_ver_ variable to the new version of your container.
+  - Change the _force_deployment_ variable to _true_.
+
+- Initialise terraform for the appropriate environment config file _backend-dev.tfbackend_ or _backend-prod.tfbackend_ run:
+
+  ```bash
+  terraform init -backend-config=env/dev/backend-dev.tfbackend -reconfigure
+  ```
+
+  The reconfigure options ensures that the backend state is reconfigured to point to the appropriate S3 bucket.
+
+- Refresh the local state to ensure it is in sync with the backend
+
+  ```bash
+  terraform refresh -var-file=env/dev/dev.tfvars
+  ```
+
+- Plan the changes, ensuring you use the correct environment config (depending upon which env you are configuring):
+
+  E.g. for the dev environment run
+
+  ```bash
+  terraform plan -var-file=env/dev/dev.tfvars
+  ```
+
+- Apply the changes, ensuring you use the correct environment config (depending upon which env you are configuring):
+
+  E.g. for the dev environment run
+
+  ```bash
+  terraform apply -var-file=env/dev/dev.tfvars
+  ```
+
+- When the terraform has applied successfully the running task will have been replaced by a task running the container version you specified in the tfvars file
 
```

Comment:
Move the content to its own section. (or delete it as it won't be needed in a future version)